lnplus/umbrel-app.yml

```diff
@@ -0,0 +1,24 @@
+manifestVersion: 1
+id: lnplus
+category: Lightning Node Management
+name: Lightning Network+
+version: "0.1.11"
+tagline: Get incoming liquidity and double your channels for free
+description: >-
+  Lightning Network Plus (LN+) is a free app that helps lightning node operators to team up and create liquidity swaps. We will walk you through each step to make it an easy process. You will get incoming liquidity, and will be able to grow your channel count and capacity twice as fast, without any extra costs. The app will automatically log you in with your node. You can join a swap or you can create a new one. You can chat with other node operators. LN+ will help you open the channel with a single click, or you can open the channel manually with the app of your choice. You can rate swap participants, and you will receive ratings to build your node's reputation.
```

Comment:
Perhaps consider some new lines here to head to users more easily read this within the Umbrel UI.